backend/routes/passRoutes.js

```diff
@@ -0,0 +1,15 @@
+const express = require("express")
+const router = express.Router()
+const { protect, authorize } = require("../middleware/authMiddleware")
```

Comment:
Do we have protect middleware?

Reply:
no, forgot to remove it while pushing the code...done now